### PR TITLE
NO-JIRA: Merge dev to events-discovery

### PR DESCRIPTION
**Pull request by stfurkan**
# READY FOR REVIEW

# Summary
- Merge `dev` to `events-discovery`

# Review By (Date)
- Soon

# Criticality
- 5

# Review Tasks

## Setup tasks and/or behavior to test

1. Make sure that the new workflow merge `dev` branch to `events-discovery` after each successful PR merge

### CODE REVIEW

File: .github/workflows/merge-to-events-discovery.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+jobs:
+  merge-to-events-discovery:
+    name: Merge to events-discovery
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    # Only run on merged PRs.
+    if: ${{ github.event.pull_request.merged }}
+
+    steps:
+      - name: Merge dev into uat
```
Comment (sherakama) [inline .github/workflows/merge-to-events-discovery.yml:18]:
```suggestion
      - name: Merge dev into events-discovery
```

**Review by sherakama — APPROVED** [review]
Good idea

File: .github/workflows/merge-to-events-discovery.yml (hunk 1 above)
Comment (stfurkan) [reply .github/workflows/merge-to-events-discovery.yml]: @sherakama I forgot to change that :D Thanks for catching it

**Comment by stfurkan** [comment]
> Good idea

Thanks for the review @sherakama :)

**Pull request merged**